servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -179,18 +262,28 @@ def _auth_token():
     return None
 
 
-def _client():
-    return current_app.config['CLIENT']
+def _get_offset_id_index(offset_id, sorted_jobs):
+    job_ids = [j['job-id'] + j.get('task-id', '') for j in sorted_jobs]
```

Comment:
Extract to a helper? This job ID key is used in multiple places.

Reply:
Done